### ISSUE TITLE
Accessibility issues with p-editor

### ISSUE THREAD

**Issue by T-jain1995** [primefaces/primeng#8949]
**I'm submitting a ...**  (check one with "x")
```
[x] bug report => Search github for a similar issue or PR before submitting
[ ] feature request => Please check if request is not on the roadmap already https://github.com/primefaces/primeng/wiki/Roadmap
[ ] support request => Please do not submit support request here, instead see http://forum.primefaces.org/viewforum.php?f=35
```

**Current behavior**
 Use p-editor in your project and run accessibility insight tool or screen reader.  We will get "Ensures buttons have discernible text".

Element does not have inner text that is visible to screen readers

**Expected behavior**
Accessibility report from Accessibility Insights for Web in Chrome should be clean.

**Minimal reproduction of the problem with instructions**

- Create an app with a p-editor component or even use the PrimeNG component sample page

- Run accessibility Insights for Web tool on your browser.

You can see multiple errors on the editor component.
![image](https://user-images.githubusercontent.com/22454085/84716127-f2fbe500-af8f-11ea-971c-b54c5e554bc4.png)

**What is the motivation / use case for changing the behavior?**

This will be helpful by the specially-abled people.
**Please tell us about your environment:**

Windows, NPM, Angular, VSCode

* **Angular version:** 9.1.9

* **PrimeNG version:** 9.0.6

* **Browser:** [Chrome, Edge ]

 
* **Language:** [all | TypeScript 3.8.3]

* **Node (for AoT issues):** `node --version` =

**Comment by mertsincan** [comment]
Hi,

So sorry for the delayed response! Improvements have been made to many components recently, both in terms of performance and enhancement. Therefore, this improvement may have been developed in another issue ticket without realizing it. You can check this in the documentation. If there is no improvement on this, can you reopen the issue so we can include it in our roadmap? 
Please don't forget to add your feedback as a comment after reopening the issue. These will be taken into account by us and will contribute to the development of this feature. Thanks a lot for your understanding!

Best Regards,
